.travis.yml

```diff
@@ -1,16 +1,12 @@
-dist: trusty
 language: python
 python:
-  - "2.7"
+  - 2.7
```

Comment:
why are we still continuing to support python 2?

Reply:
It is just for the transition to python 3.8 Since currently it just works for pyhton 2.7, we want to support python 2.7 and 3.5 in this PR and test that everything is working fine. Then in other PR we can drop support for python 2.7. Similar es here https://github.com/edx/notifier/pull/82.